daemon/volumes.go

```diff
@@ -119,7 +119,15 @@ func (daemon *Daemon) registerMountPoints(container *container.Container, hostCo
 		}
 	}
 
-	// 3. Read bind mounts
+	// 3. Read tmpfs
+	for destination := range hostConfig.Tmpfs {
+		mountPoints[destination] = &volume.MountPoint{
+			Source:      "tmpfs",
```

Comment:
it isn't actually that bad to have "Source == tmpfs" since it could be saying this volume source comes from a tmpfs (I'm joking was just trying to make an argument)